### PR TITLE
Added Player and Roles classes

### PR DESCRIPTION
**Pull request by Ji-Justin**
Added Player, Roles and basic roles classes.

### CODE REVIEW

File: src/main/java/com/example/TwoRoomsBoom/models/Player.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+public class Player {
+    private String name;
+    private Role role;
+    private Set<String> conditions;
+    private Integer room;
+    @Default
+    private boolean isLeader = false;
+    private boolean isHost;
+    
+    public void changeRoom() {
```
Comment (RileyChampion) [inline src/main/java/com/example/TwoRoomsBoom/models/Player.java:26]: Thinking about this now, should the player have this action? Or should the Room Class be in control of changing the rooms of players?

File: src/main/java/com/example/TwoRoomsBoom/models/role/BlueTeam.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.example.TwoRoomsBoom.models.role;
+
+import com.example.TwoRoomsBoom.models.Role;
+
+public class BlueTeam extends Role{
```
Comment (RileyChampion) [inline src/main/java/com/example/TwoRoomsBoom/models/role/BlueTeam.java:5]: Just so I know, are the normal team players just called Blue and Red team? Are they not called like agents or anything? Just curious

**Review by RileyChampion — COMMENTED** [review]
Some minor comments

File: src/main/java/com/example/TwoRoomsBoom/models/Player.java (hunk 1 above)
Comment (Ji-Justin) [reply src/main/java/com/example/TwoRoomsBoom/models/Player.java:26]: Good point, Room Class should probably handle exchanging players, not the Player Class.

File: src/main/java/com/example/TwoRoomsBoom/models/role/BlueTeam.java (hunk 2 above)
Comment (Ji-Justin) [reply src/main/java/com/example/TwoRoomsBoom/models/role/BlueTeam.java:5]: Yes, the normal team players are just called Blue and Red team. Agents are completely different roles.

**Review by RileyChampion — APPROVED** [review]

**Pull request merged**